### DISCUSSION TITLE
Datalinq Auswahllisten im App-Builder

### DISCUSSION THREAD

**Discussion by thomaskofler** [e-netze/webgis-community#166] [General]
Hi Jürgen,

wenn man im CMS beim Editing Auswahllisten vom Datalinq (plaintext) verwendet und dann den Edit-Layer im App-Builder verwenden möchte, kommt eine Fehlermeldung (habe dann die Auswahlliste wieder entfernt, dann ging auch der Edit-Layer):

![grafik](https://github.com/gis-eni/webgis-forum/assets/119412172/81259162-188e-46e1-90b0-9335d74db046)

Wird der gleiche Edit-Layer in einer Karte aufgerufen funktionieren die Auswahllisten wie gewohnt.

**Comment by gis-eni** [comment]
Hi Thomas,

ja, diese Auswahllisten sind bei AppBuilder Apps bisher nicht supported gewesen. Du musst den letzten Build 5.24.1101 installieren. Damit sollte es gehen.

**Reply by thomaskofler** [reply]
Optimal danke :)
